content/intro-to-storybook/angular/en/test.md

```diff
@@ -104,14 +104,14 @@ If a change is intentional we'll need to update the baseline so that future test
   />
 </video>
 
-Since modern apps are constructed from components, it’s important that we test at the level of component. Doing so helps us pinpoint the root cause of a change, the component, instead of reacting to symptoms of a change, the screens and composite components.
+Since modern apps are constructed from components, it’s important that we test at the level of the component. Doing so helps us pinpoint the root cause of a change, the component, instead of reacting to symptoms of a change, the screens, and composite components.
```

Comment:
```suggestion
Since modern apps are constructed from components, it’s important that we test at the level of the component. Doing so helps us pinpoint the root cause of a change, the component, instead of reacting to symptoms of a change: the screens and composite components.
```

It was correct before; this suggestion makes it more clear.

Reply:
Applied the suggestion, thanks for the clarification